s3upload.js

```diff
@@ -60,7 +66,7 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
-    var fileName = file.name.replace(/\s+/g, "_");
+    var fileName = file.name.latinize();
```

Comment:
I think we still want the whitespace to underscore conversion.  It doesn't look like `latinize()` will handle that, so can we add that back?